File: frontend/src/app/dto/warehouse/CreateWarehouseDTO.ts
export interface CreateWarehouseDTO {
  name: string;
  countryId: number;
  cityId: number;
  street: string;
  streetNumber: string;
  latitude: number;
  longitude: number;
  totalCapacity: number;
  sectors: CreateWarehouseSectorDTO[];
}

export interface CreateWarehouseSectorDTO {
  name: string;
  minTemperature: number;
  maxTemperature: number;
  capacity: number;
}

File: backend/src/main/java/nvt/backend/dto/warehouse/CreateWarehouseDTO.java
package nvt.backend.dto.warehouse;

import lombok.Data;

import java.util.List;

@Data
public class CreateWarehouseDTO {
    private String name;
    private Long countryId;
    private Long cityId;
    private String street;
    private String streetNumber;
    private Double latitude;
    private Double longitude;
    private Double totalCapacity;
    private List<CreateWarehouseSectorDTO> sectors;
}
